src/Paket.Core/PaketConfigFiles/ProjectFile.fs

```diff
@@ -1934,6 +1934,10 @@ type ProjectFile with
             search true (DirectoryInfo folder)
 
         findAllFiles folder
+
+    /// Finds all project files
+    static member FindAllProjectFiles folder : FileInfo [] =
+        ProjectFile.SearchAllProjectRelatedFiles "*proj*" folder
```

Comment:
that's not the same pattern as we did above, right? This would include "MyprojectFile.fs" or stuff like that

Reply:
This is the same as what it was originally (see L1902). Yes it would, but it gets filtered out anyway in the next line.

Do you think I should change it to `"*.*proj"`?